src/lib/containers/SynapseTable.tsx

```diff
@@ -184,11 +183,15 @@ export default class SynapseTable extends React.Component<QueryWrapperChildProps
         <button onClick={this.closeMenuClickHandler} className={`SRC-menu-wall ${optionalHiddenClass}`} />
         <div className="container-fluid">
             <div className="row SRC-marginBottomTen">
-                <span>
-                    {!isLoading && <strong> Showing {total} {unitDescription} </strong>}
-                    <span className={isLoading ? 'spinner' : ''} style={isLoading ? {} : { display: 'none' }} />
-                    {isLoading && <strong> {'    '} Table results updating... </strong>}
-                </span>
+              <p style={{ height:'20px' }}>
```

Comment:
this makes the spinner icon come into the page without making the element expand unnaturally